lib/srv/db/proxyserver.go

```diff
@@ -24,6 +24,10 @@ import (
 	"io"
 	"net"
 	"strings"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/sirupsen/logrus"
```

Comment:
Nit: We follow [GoCodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments) which suggests grouping all imports in 3 groups: first stdlib, then imports from same package (`github.com/g11l/teleport`) and then 3rd party imports. So I'd move this to the 3rd group (where trace is below). If you install `goimports` it should do this automatically (most of the time).

Reply:
thanks